playwright/tests/customers.spec.ts

```diff
@@ -0,0 +1,226 @@
+  await customersPage.saveCustomer();
+  await customersPage.expectSuccessBanner();
+  await expect(customersPage.pageHeader).toContainText(`${firstName} ${lastName}`);
+  await expect(customersPage.customerNoteInput).toContainText(note);
+  await expect(customersPage.customerFirstNameInput).toHaveValue(firstName);
+  await expect(customersPage.customerLastNameInput).toHaveValue(lastName);
+  await expect(customersPage.customerEmailInput).toHaveValue(email.toLowerCase());
+});
+test("TC: SALEOR_200 As an admin I should not be able to create customer with duplicated email @e2e @customer", async () => {
+  const firstName = faker.name.firstName();
+  const lastName = faker.name.lastName();
+  const note = faker.lorem.sentence();
+
+  await customersPage.goToCustomersListView();
+  await customersPage.clickOnCreateCustomer();
+  await customersPage.fillFirstAndLastName(firstName, lastName);
+  await customersPage.fillEmail(CUSTOMERS.customerToBeDeactivated.email);
+
+  const newAddress = ADDRESS.addressUS;
+
+  await addressForm.completeBasicInfoAddressForm(newAddress);
+  await addressForm.typeCompanyName(newAddress.companyName);
+  await addressForm.typePhone(newAddress.phone);
+  await addressForm.typeAddressLine2(newAddress.addressLine2);
+  await addressForm.selectCountryArea(newAddress.countryArea);
+  await customersPage.fillNote(note);
+  await customersPage.saveCustomer();
+  await customersPage.expectErrorBannerMessage("User with this Email already exists.");
+});
+
+test("TC: SALEOR_201 Update customer account info @e2e @customer", async () => {
+  const firstName = faker.name.firstName();
+  const lastName = faker.name.lastName();
+  const email = faker.internet.email();
+  const note = faker.lorem.sentence();
+
+  await customersPage.gotoCustomerDetailsPage(CUSTOMERS.editCustomer.id);
+  await customersPage.fillNote(note);
+  await customersPage.fillFirstAndLastName(firstName, lastName);
+  await customersPage.fillEmail(email);
+  await customersPage.saveCustomer();
+  await customersPage.expectSuccessBanner();
+  await expect(customersPage.pageHeader).toContainText(`${firstName} ${lastName}`);
+  await expect(customersPage.customerNoteInput).toContainText(note);
+  await expect(customersPage.customerFirstNameInput).toHaveValue(firstName);
+  await expect(customersPage.customerLastNameInput).toHaveValue(lastName);
+  await expect(customersPage.customerEmailInput).toHaveValue(email.toLowerCase());
+});
+test("TC: SALEOR_202 Deactivate a customer @e2e @customer", async () => {
+  await customersPage.gotoCustomerDetailsPage(CUSTOMERS.customerToBeDeactivated.id);
+  await customersPage.customerActiveCheckbox.click();
+  await customersPage.saveCustomer();
+  await customersPage.expectSuccessBanner();
+  await expect(customersPage.customerActiveCheckbox).not.toBeChecked();
+});
+test("TC: SALEOR_203 Activate a customer @e2e @customer", async () => {
+  customersPage.gotoCustomerDetailsPage(CUSTOMERS.customerToBeActivated.id),
```

Comment:
Missing await and `,` in the end of line

Reply:
This i perfect case for use [eslint-plugin-playwright](https://www.npmjs.com/package/eslint-plugin-playwright)

Reply:
fixed